events/wc_realm_events.txt

```diff
@@ -545,4 +545,158 @@ character_event = {
 		# Clear flags and other stuff
 		clear_settler_flag_effect = yes
 	}
+}
+
+### Ymiron Awakened ###
+character_event = {
+	id = WCRLM.799
+	
+	is_triggered_only = yes
+	hide_window = yes
+	
+	immediate = {
+		# Saves ROOT's liege for future use
+		if = {
+			limit = { independent = no }
+			liege = { save_event_target_as = target_liege }
+		}
+		
+		# Creates Ymiron
+		create_character = {
+			name=Ymiron
+			dynasty=70000
+			dna="anolmcpmehb"
+			culture=vrykul religion=ROOT
+			attributes = { martial=6 diplomacy=6 stewardship=4 intrigue=7 learning=7 }
+			trait=skilled_tactician
+			trait=class_warrior_7
+			trait=wroth trait=proud trait=ambitious trait=arbitrary trait=paranoid
+			trait=creature_vrykul
+			random_traits = yes
+			age = 300
+		}
+		new_character = {
+			save_event_target_as = target_ymiron
+		}
+		event_target:target_ymiron = {
+			# Gives bloodline
+			create_bloodline = {
+				type = ymiron
+			}
+			
+			# Angerboda becomes your spouse
+			c_100001 = {
```

Comment:
Will marry even if this character can't marry

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/71768853-728a8000-2f2b-11ea-84ab-06738f7fcbeb.png)
![image](https://user-images.githubusercontent.com/46576860/71768862-7ae2bb00-2f2b-11ea-8b89-f3505cf5b800.png)
![image](https://user-images.githubusercontent.com/46576860/71768866-81713280-2f2b-11ea-8f30-c4c8e95f714d.png)

</details>

Reply:
Fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/280aecdd7daba1dec5ad0f59bddc472cf2d9bc47.